plugin-server/src/worker/ingestion/team-manager.ts

```diff
@@ -138,13 +137,13 @@ export class TeamManager {
             this.eventNamesCache.get(team.id)?.add(event)
         } else {
             // TODO: #7422 Temporary conditional to test experimental feature
-            if (this.experimentalLastSeenAtEnabledTeams.includes(team.id)) {
+            if (this.experimentalLastSeenAtEnabled) {
                 const eventCacheKey = JSON.stringify([team.id, event])
                 if ((this.eventLastSeenCache.get(eventCacheKey) ?? 0) < eventTimestamp.valueOf()) {
                     this.eventLastSeenCache.set(eventCacheKey, eventTimestamp.valueOf())
                 }
                 // TODO: Allow configuring this via env vars
-                if (this.eventLastSeenCache.size > 1000 || DateTime.now().diff(this.lastFlushAt).as('seconds') > 60) {
+                if (this.eventLastSeenCache.size > 1000 || DateTime.now().diff(this.lastFlushAt).as('seconds') > 120) {
```

Comment:
I'd add a note here that the `120` here is because we flush every `60` in the main thread.